src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/PairedStrandedIntervals.java

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+public final class PairedStrandedIntervals {
```

Comment:
I think it might be helpful to have this class more clearly model an uncertain breakpoint, rather than just a pair of intervals with strand.  As it stands, it's not clear what the relationship between the two sequences is.  I think I've figured out that in the existing code the two sequences are extensions of the alignments of a read pair -- and therefore expected to be on opposite strands.
So the data held by this class is fine, but the interpretation of the relationship between left and right could be made simpler and more explicit, I think.
I'd like to see a head sequence and a tail sequence (rather than a left and right) with the strandedness manipulated so that the actual sample sequence over the breakpoint -- if we were able to figure it out exactly -- would be some prefix of the head sequence followed by some suffix of the tail sequence.
This is probably totally unclear, and we can chat.